scripts/pyenv_setup.sh

```diff
@@ -116,7 +117,10 @@ setup_pyenv() {
   # If the script is called with the "dot space right" approach (. ./scripts/pyenv_setup.sh),
   # the effects of this will be persistent outside of this script
   echo "Activating pyenv and validating Python version"
+  # Sets up PATH
   eval "$(pyenv init --path)"
+  # Enables autocompletion and all subcommands
+  eval "$(pyenv init -)"
```

Comment:
Remove this as this will not persist outside of the `make setup-pyenv` call.
The one above we need because, otherwise, `which python` will default to `/usr/bin/python`.